### PR TITLE
Lookup tournament player statistics for the correct ruleset

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on https://github.com/ppy/osu-framework/pull/2976

### CODE REVIEW

**Review by smoogipoo — DISMISSED** [review]

**Comment by smoogipoo** [comment]
Requires conflict resolution

File: osu.Game/Graphics/UserInterfaceV2/LabelledDropdown.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Graphics.UserInterfaceV2
+{
+    public class LabelledDropdown<T> : LabelledComponent<OsuDropdown<T>, T>
```
Comment (smoogipoo) [inline osu.Game/Graphics/UserInterfaceV2/LabelledDropdown.cs]: This should be split out into its own PR because it feels pretty bad atm: https://streamable.com/0s7js

Comment (peppy) [reply osu.Game/Graphics/UserInterfaceV2/LabelledDropdown.cs]: Will move local for now, the fix for this isn't so straightforward.

**Review by smoogipoo — DISMISSED** [review]

**Review by smoogipoo — APPROVED** [review]
